package.json

```diff
@@ -25,7 +25,9 @@
     "onLanguage:xml",
     "onLanguage:xsl",
     "onCommand:xml.open.docs",
-    "onCommand:xml.open.docs.home"
+    "onCommand:xml.open.docs.home",
+    "onCommand:xml.validation.current.file",
```

Comment:
No need to add these here, since it doesn't make sense to try to validate a file before opening it.

Reply:
fixed